deeplake/client/client.py

```diff
@@ -188,15 +188,11 @@ def request_auth_token(self, username: str, password: str):
             LoginException: If there is an issue retrieving the auth token.
 
         """
-        json = {"username": username, "password": password}
-        response = self.request("POST", GET_TOKEN_SUFFIX, json=json)
```

Comment:
shell we keep `"post" GET_TOKEN_SUFFIX` in DeepLakeClient?

Reply:
Shouldn't need that anymore, removing it